app/views/desks/index.html.erb

```diff
@@ -1,7 +1,43 @@
-<h3>Desk list</h3>
-<% @desks.each do |desk| %>
-  <ul>
-    <li><%= link_to desk.name, desk_path(desk) %></li>
-    <li id=<%="desk_state_#{desk.id}" %>><%= desk.occupied %></li>
-  </ul>
+<div class="alert alert-warning alert-dismissible fade show" role="alert">
+  <strong>Click on the status to see all desk details!</strong>
+  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>
+
+<br>
+
+<div class="row desk-row">
+<% @desks.sort().each do |desk| %>
```

Comment:
I'd order them in SQL land in `desks_controller` so that we keep the view as dumb as possible and as performant as possible since SQL it's a lot faster than ruby for when you have lots of items.

https://github.com/Smart-Library/smartlibrary/blob/97268a93c469c585b2f80d4b52beeaa0666e971c/app/controllers/desks_controller.rb#L7-L9

If you want to order them by `id` you can do `Desk.order(:id)`